x-pack/plugins/cross_cluster_replication/public/app/components/section_error.js

```diff
@@ -11,21 +11,19 @@ import { EuiCallOut, EuiSpacer } from '@elastic/eui';
 export function SectionError(props) {
```

Comment:
I'll migrate this to use our shared `SectionError` as part of https://github.com/elastic/kibana/issues/84801